apps/editors/helpers.py

```diff
@@ -807,7 +806,8 @@ class ReviewFiles(ReviewBase):
 
     def set_data(self, data):
         self.data = data
-        self.files = data.get('addon_files', None)
+        if 'addon_files' in data:
+            self.files = data['addon_files']
```

Comment:
For a "files" review, we might want to only review one file. That's not possible using the forms (using the editor tools), but for example the `approve_addons` management command and the `auto_sign_file` helpers only sign a file at a time.